doc/remote.md

```diff
@@ -0,0 +1,37 @@
+## Run Lightning Terminal with a remote LND instance
+To connect Lightning Terminal to a remote LND instance first make sure your `lnd.conf` file contains the following additional configuration settings:
+```
+tlsextraip=<externally-reachable-ip-address>
+rpclisten=0.0.0.0:10009
+```
+
+Copy the following files that are located in your `~/.lnd/data/chain/bitcoin/mainnet` directory on your remote machine to `/some/folder/with/lnd/data/` on your local machine (where you’ll be running LiT):
+- tls.cert
+- admin.macaroon
+- chainnotifier.macaroon
```

Comment:
Only the `admin.macaroon` file is needed. They do not need to copy over all of the macaroons.

Reply:
I had assumed I just needed tls.cert and admin.macaroon but when I actually went to run it I was prompted for the other macaroons. I'll do more digging

Reply:
Yeah, loop and faraday need all macaroons, unfortunately.